.github/workflows/master.yml

```diff
@@ -1,23 +0,0 @@
-name: CI
-
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
-
-  workflow_dispatch:
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-
-    steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v3
-
-      - name: Install shellcheck
-        run: sudo apt-get install -qq shellcheck
-
-      - name: Lint shell scripts
-        run: shellcheck $GITHUB_WORKSPACE/*.sh
```

Comment:
Was this check merged into the Jenkins pipeline?

Reply:
I added this in Jenkins pipeline in this PR https://github.com/brave/devops/pull/8064/files#diff-016e64047994aa44889af773c7791164370c71bcc0a068c69e22a8f282360096R29, waiting for your approval for this one, so I can merge them together